lib/feature/add_character/add_character_page.dart

```diff
@@ -0,0 +1,204 @@
+            SnackBar(
+              content: Text('Character added: ${state.newCharacter.title}'),
+              backgroundColor: Colors.green,
+            ),
+          );
+        }
+      },
+      builder: (context, state) {
+        return Scaffold(
+          body: _buildUI(context, state),
+        );
+      },
+    );
+  }
+
+  Widget _buildUI(BuildContext context, AddCharacterState state) {
+    if (state is AddCharacterStateInitial) {
+      return _buildInitialUI(context);
+    } else if (state is AddCharacterStateSuccess) {
+      return _buildSuccessUI(context, state.newCharacter);
+    } else if (state is AddCharacterStateFailure) {
+      return _buildErrorUI(context, state.error);
+    } else {
+      return const Center(
+        child: CircularProgressIndicator(),
+      );
+    }
+  }
+
+  Widget _buildInitialUI(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(16.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          _buildNeumorphicTextField(titleController, 'Character Title'),
+          const SizedBox(height: 20),
+          _buildNeumorphicTextField(imageUrlController, 'Image URL'),
+          const SizedBox(height: 20),
+          _buildNeumorphicTextField(
+            descriptionController,
+            'Character Description',
+          ),
+          const SizedBox(height: 20),
+          ElevatedButton(
+            onPressed: () {
+              final title = titleController.text.trim();
+              final imageUrl = imageUrlController.text.trim();
+              final description = descriptionController.text.trim();
+
+              if (title.isNotEmpty &&
+                  imageUrl.isNotEmpty &&
+                  description.isNotEmpty) {
+                context.read<AddCharacterCubit>().addCharacter(
+                      title: title,
+                      imageUrl: imageUrl,
+                      description: description,
+                    );
+              } else {
+                ScaffoldMessenger.of(context).showSnackBar(
+                  const SnackBar(
+                    content: Text('All fields must be filled'),
+                    backgroundColor: Colors.red,
+                  ),
+                );
+              }
+            },
+            style: ElevatedButton.styleFrom(
+              primary: Colors.grey[200], // фон кнопки
+              onPrimary: Colors.grey[800], // цвет текста кнопки
+              elevation: 8,
+              shape: RoundedRectangleBorder(
+                borderRadius: BorderRadius.circular(16),
+              ),
+            ),
+            child: const Text('Add Character'),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildNeumorphicTextField(
+      TextEditingController controller, String labelText) {
+    return Container(
+      decoration: BoxDecoration(
+        color: Colors.grey[200], // Цвет фона текстового поля
```

Comment:
wow, I don't think that we should use Russian to write some comments in code. Please remove it or use English

Reply:
Done